src/base/PackableStream.h

```diff
@@ -13,20 +13,18 @@
 
 #include <ostream>
 
-/**
- * Provides a streambuf interface for writing to Packable objects.
- * Typical use is via a PackableStream rather than direct manipulation
- */
-class PackableStreamBuf : public std::streambuf
+// TODO: Move to src/base/AppendingStreamBuf.h
```

Comment:
This TODO can be in this PR scope. I did not create src/base/AppendingStreamBuf.h because it would hide/obscure code changes. I suggest doing that last step after the PR is otherwise ready for merging.